Frontend/src/components/Admin/Users/AdminListUsers.tsx

```diff
@@ -14,14 +14,16 @@ export default () => {
     const [toDelete, setToDelete] = useState<IUser | null>(null);
     const [info, setInfo] = useState<string | null>(null);
     const [error, setError] = useState<string | null>(null);
+    const [search, setSearch] = useState("");
```

Comment:
Type is missing

Reply:
The type is inferred in this case, it can only be a string.

In the cases above, I had to force it to be a certain type.